FreeRTOS_RA.c

```diff
@@ -461,9 +461,9 @@
 
                    /* Send a Router Solicitation to ff02::2 */
                    ( void ) memset( xIPAddress.ucBytes, 0, sizeof xIPAddress.ucBytes );
-                   xIPAddress.ucBytes[ 0 ] = 0xff;
-                   xIPAddress.ucBytes[ 1 ] = 0x02;
-                   xIPAddress.ucBytes[ 15 ] = 0x02;
+                   xIPAddress.ucBytes[ 0 ] = 0xffU;
+                   xIPAddress.ucBytes[ 1 ] = 0x02U;
+                   xIPAddress.ucBytes[ 15 ] = 0x02U;
```

Comment:
I don't think so but still confirming: Are these values worthy of being converted to a macro?